hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -119,7 +118,7 @@ public static void clearGlobalProps() {
    *
    * @param filePath File path for configuration file
    */
-  public void addPropsFromFile(Path filePath) {
+  public void addPropsFromFile(Path filePath, boolean isDefaultDirectory) {
```

Comment:
don't think we need extra param for this. if `filePath` is the default one, we know it by comparing it with `new Path(DEFAULT_CONF_FILE_DIR + "/" + DEFAULT_PROPERTIES_FILE)` ?

Reply:
+1.  `visitedFilePaths` also stores the loaded props file so each props file is only read once.